scripts/ai-help-macros.ts

```diff
@@ -142,20 +158,38 @@ export async function updateEmbeddings(
         markdown,
         markdown_hash,
       });
+    } else if (!existingDoc.has_embedding || !existingDoc.has_embedding_next) {
```

Comment:
The `else if` here doesn't look right. This should still get executed even if the `markdown_hash` changed.

Reply:
Good catch, fixed in [61b1547](https://github.com/mdn/yari/pull/10818/commits/61b154791d4ac54c00a16cca7c8e0dd1b71673ba).